transport/http/do.go

```diff
@@ -0,0 +1,90 @@
+package http
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"go.uber.org/yarpc/api/transport"
+	intyarpcerrors "go.uber.org/yarpc/internal/yarpcerrors"
+	"go.uber.org/yarpc/yarpcerrors"
+)
+
+//RoundTrip implements the http.RoundTripper interface, making a YARPC HTTP outbound suitable as a
+// Transport when constructing an HTTP Client. An HTTP client is suitable only for relative paths to
+// a single outbound service.
+//
+//The HTTP outbound refuses to send HTTP requests that have a fully qualified path, since it cannot
+// respect the host and protocol portions of the URL, instead routing through the outbound peer
+// chooser. A request that specifies a host or protocol will return an error.
+//
+// Sample usage:
+//
+// client := http.Client{Transport: outbound}
```

Comment:
Indent example one space deeper and it’ll render as a code block.

Reply:
Missed this line. Will address in the next push.